address/address.go

```diff
@@ -0,0 +1,61 @@
+package address
```

Comment:
What do you think about putting this package inside an `internal` package to avoid publicly exposing this functionality? I'm assuming there are duplicate implementations in go-algorand and go-algorand-sdk, so anyone consuming this library should probably use their own implementation instead of this one.

Reply:
That's an option, but since we have to define an implementation here I think we might as well export it.

My hidden agenda is to make go-algorand-sdk drop its implementation in favor of this once, since it already has to import this repo.